Update maas docs (#69)
voi-api/voi-api.github.io: maas-pro.md

The `beginners mode` is a feature that limits the speed of the vehicle to the zone reduced speed. This is a feature that is used to make the scooter more accessible to new users.

<aside class="notice">

> It is upto the partner to decide if the user reached the minimum number of required rides to be able to start a rental without beginners mode.

Reviewer: ```suggestion
It is up to the partner to decide if the user reached the minimum number of required rides to be able to start a rental without beginner's mode.
```

</aside>

<aside class="notice">

> We do not limit the duration of the rental as soon as its running. 

Reviewer: ```suggestion
We do not restrict the duration of any rental while it is active. 
```
